test/error.js

```diff
@@ -204,7 +204,10 @@ test('error.code is undefined on success', async t => {
 	t.is(code, undefined);
 });
 
-test('error.code is defined on failure if applicable', async t => {
-	const {code} = await t.throwsAsync(execa('invalid'));
-	t.is(code, 'ENOENT');
-});
+// Windows doesn't throw ENOENT on invalid command
+if (process.platform !== 'win32') {
+	test('error.code is defined on failure if applicable', async t => {
+		const {code} = await t.throwsAsync(execa('invalid'));
+		t.is(code, 'ENOENT');
```

Comment:
There might be ways to fix this test in a way that works on Windows as well. 
We need to find an `error.code` that is cross-platform.

The following might work:

```js
test('error.code is defined on failure if applicable', async t => {
	const {code} = await t.throwsAsync(execa(''));
	t.is(code, 'ERR_INVALID_ARG_VALUE');
});
```

Reply:
Unfortunately, using an empty command on Windows doesn't generate `ERR_INVALID_ARG_VALUE`. There aren't many errors thrown consistently across both POSIX & Windows and also across multiple Node versions. The one I found that seems to work is giving an incorrect type for `cwd`. So I used that again here in the updated version.

Reply:
That works :+1: